package.json

```diff
@@ -39,4 +39,4 @@
     "stryker": "stryker run",
     "test": "./node_modules/.bin/mocha -R spec"
   }
-}
+}
```

Comment:
And finally, make sure lines are terminated with a newline. Git has trouble with lines that are missing a newline character at the end.